src/Magento/FunctionalTestingFramework/Suite/views/SuiteClass.mustache

```diff
@@ -84,6 +88,9 @@ class {{suiteName}} extends \Codeception\GroupObject
         if ($this->currentTestRun == $this->testCount) {
             print sprintf(self::$HOOK_EXECUTION_INIT, "after");
 
+            /** @var MagentoWebDriver $webDriver */
+            $webDriver = $this->getModule('\Magento\FunctionalTestingFramework\Module\MagentoWebDriver');
```

Comment:
I'd rather we use `MagentoWebDriver::class` if possible (here and all other places in this PR).